.env

```diff
@@ -9,13 +9,13 @@ MONGO_URL="mongodb://simplemongousername:simplemongopassword@localhost:27030/?au
 # development
 # ST_URL="http://127.0.0.1:8084" 
 # production
-ST_URL="https://darpaitm.caci.com/soartech" 
+ST_URL="http://10.216.38.125:8084"
 
 # location of adept server
  # development
 # ADEPT_URL="http://127.0.0.1:8080"
 # production
-ADEPT_URL="https://darpaitm.caci.com/adept" 
+ADEPT_URL="http://10.216.38.70:8080/"
```

Comment:
Shouldn't this not have a trailing slash?

Reply:
The script didn't run without it.